lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -51,6 +51,7 @@ public static function get($ids, $display = 'teaser') {
     $results = node_load_multiple($ids);
 
     if (!$results) {
+      http_response_code(404);
       throw new Exception('No campaign data found.');
```

Comment:
@aaronschachter if we just `return http_response_code(404);` here's what will get returned: 

```
{
  data: {
    id: null,
    title: null, 
    campaign_runs: null,
    language: null,
    translations: null,
    tagline: null,
    status: null,
    type: null
  }
}
```

would we want the above or for the error response to be returned?